### ISSUE TITLE
Error with logrotate

### ISSUE THREAD

**Issue by n-rodriguez** [saltstack-formulas/bind-formula#83]
Hi!

I get an error with this logrotate :

```
/var/log/bind9/query.log {
  rotate 7
  daily
  missingok
  notifempty
  sharedscripts
  copytruncate
  compress
  create 0664 bind root
  su
}
```

```
switching euid to -1 and egid to -1
error: error switching euid to -1 and egid to -1: Invalid argument
```

I think we should pass user and group to the `su` directive : `su bind bind`.

Thanks!

**Comment by javierbertoli** [comment]
@n-rodriguez I can't reproduce this error. Any other info you might share? Distro? packages versions (logrotate, bind9, etc?)

**Comment by javierbertoli** [comment]
Fixed on #89
